tests/integration/cmd_debug_test.go

```diff
@@ -55,9 +56,11 @@ var _ = Describe("odo debug command tests", func() {
 			// Make sure that the debug information output, outputs correctly.
 			// We do *not* check the json output since the debugProcessID will be different each time.
 			helper.WaitForCmdOut("odo", []string{"debug", "info", "--context", context, "-o", "json"}, 1, true, func(output string) bool {
-				Expect(output).To(ContainSubstring(`"kind": "OdoDebugInfo"`))
-				Expect(output).To(ContainSubstring(`"localPort": ` + freePort))
-				return true
+				if strings.Contains(output, `"kind": "OdoDebugInfo"`) &&
```

Comment:
Awesome!!!
You saved a coming soon flake...hahaha...

@mik-dass Thanks a lot 

/lgtm

Reply:
Thanks :+1: